doc/releases/index.rst

```diff
@@ -44,6 +44,22 @@ following releases:
 For more information, see  :ref:`Security Vulnerability Reporting <reporting>`.
 
 
+Supported Releases
+******************
+
++-----------------+----------------+---------------+
+| Release         | Release date   | EOL           |
++=================+================+===============+
+| `Zephyr 2.7.4`_ | 23/12/2022     | 31/08/2024    |
++-----------------+----------------+---------------+
+| `Zephyr 3.3.0`_ | 19/02/2023     | 31/10/2023    |
++-----------------+----------------+---------------+
+| `Zephyr 3.2.0`_ | 30/09/2022     | 31/06/2023    |
```

Comment:
Non-blocking: do you want to wait for @kartben to propose a method to keep this from bitrotting, or just merge and deal with it later?

Reply:
I would go with this now, and have @kartben build on top of that.

Reply:
> > I didn't get notifications about security updates

> So I think this language should be changed to "should be backported [...]" to make it clear that this is a goal and not a guarantee.

This is based on 
https://docs.zephyrproject.org/latest/security/reporting.html#backporting-of-security-vulnerabilities

where the language is "shall".